docs/semgrep-code/editor.md

```diff
@@ -215,12 +215,14 @@ To run your rule against selected repositories or projects:
 
 ## Set a rule’s visibility and share a rule
 
-Upon saving, a rule’s visibility is **unlisted** by default. This rule can be shared with anyone through an identifier, even to non-Semgrep AppSec Platform users.
+Upon saving, a rule’s visibility is **private** by default. A private rule is visible only to members within an organization.
```

Comment:
Thank you for updating this bit. I hadn't realized it had gotten so out of date.